contracts/modules/PoolController.sol

```diff
@@ -125,14 +128,27 @@ contract PoolController is
         riskpool.decreaseBalance(processId, amount);
     }
 
+    function isArchivingAllowed(uint256 id) external view returns (bool) {
+        IRiskpool riskpool = _getRiskpoolForId(id);
+        uint256 riskpoolId = riskpool.getId();
+        require(
+            _component.getComponentState(riskpoolId) == IComponent.ComponentState.Paused
+            || _component.getComponentState(riskpoolId) == IComponent.ComponentState.Suspended, 
+            "ERROR:POL-010:ARCHIVED_IS_FINAL_STATE"
```

Comment:
suggestion to rename error message to "ERROR:POL-010:TRANSITION_TO_ARCHIVED_STATE_INVALID"
motivation: ARCHIVED_IS_FINAL_STATE is true independent of context and does not provide a lot of information that helps to better understand the issue